docs/reference/ilm/apis/slm-api.asciidoc

```diff
@@ -0,0 +1,347 @@
+    } ,
+    "next_execution": "2019-04-24T01:30:00.000Z",
+    "next_execution_millis": 1556048160000
+  }
+}
+--------------------------------------------------
+// TESTRESPONSE[skip:the presence of last_failure and last_success is asynchronous and will be present for users, but is untestable]
+<1> The policy's version has been incremented because it was updated
+<2> The last successfully initiated snapshot information
+
+It is a good idea to test policies using the execute API to ensure they work.
+
+=== Delete Snapshot Lifecycle Policy API
+
+A policy can be deleted by issuing a delete request with the policy id. Note
+that this prevents any future snapshots from being taken, but does not cancel
+any currently ongoing snapshots.
```

Comment:
Or delete any snapshots previously taken by the policy. Might seem obvious, but probably best to be as clear as possible.